lib/Cake/Controller/Component/PaginatorComponent.php

```diff
@@ -225,7 +241,8 @@ public function paginate($object = null, $scope = array(), $whitelist = array())
 			'order' => $order,
 			'limit' => $limit,
 			'options' => Hash::diff($options, $defaults),
-			'paramType' => $options['paramType']
+			'paramType' => $options['paramType'],
+			'queryScope' => Hash::get($options, 'queryScope'),
```

Comment:
Please add `queryScope` to `getDefaults()`. Then you can use it here via `$options['queryScope']` instead, just for consistency.